### DISCUSSION TITLE
Hopes for 86Box?

### DISCUSSION THREAD

**Discussion by jadasse** [86Box/86Box#2402] [General]
What do you specifically have high hopes for 86Box to possibly implement in the future?

----
@dhrdlicka EDIT -- Please open feature requests as new issues (for concrete proposal, with datasheets, documentation and what not), or discussions (for abstract suggestions).

**Comment by jadasse** [comment]
for me, the Sound Blaster Live!

**Reply by Cacodemon345** [reply]
Sounds Blaster Live won't be implemented as it's too complicated.

**Reply by Cacodemon345** [reply]
That won't be enough. Even with documentation it will about as hard to implement as the original Xbox's sound hardware emulation i.e. very difficult.

**Reply by Cacodemon345** [reply]
That is not useful.

**Reply by OBattler** [reply]
The User's Manual is useless for our purposes. We need a datasheet or a technical reference.

**Reply by Cacodemon345** [reply]
Useless.

**Reply by jadasse** [reply]
> Sounds Blaster Live won't be implemented as it's too complicated.

that's a shame

**Reply by Cacodemon345** [reply]
Useless as well.

**Reply by Cacodemon345** [reply]
https://github.com/torvalds/linux/tree/master/sound/pci/emu10k1
Source code documentation (potentially partial).

**Comment by ArnoldUK** [comment]
I'm using 86Box much more now than DosBox as the hardware emulation capabilities are much more reliable for running anything other than vanilla DOS systems. One feature that I would like to see implemented is shared folder or mounting folder/disk with the host for transferring files quickly.  At the moment I open the disk image in WinImage and drag them over to the image but it's not ideal. DosBox and VirtualBox have this feature which is something I miss now using 86Box.
Overall I'm quite pleased with the hardware support on 86Box and it's ancester PCEm.

Another thing regarding new features, is not to add too much as I don't want to see 86Box become a bloated and over complicated emulation system to work with. PCEm was very simple with an nice pleasing interface but is packed with more power than VirtualBox and DosBox put together. Please DO NOT add too much glitter to the interface or integrate any powerful system mangager such as WinBox. WinBox is brilliant and the developer made it very similar to VirtualBox, but I prefer to hand configure my systems as it keeps file count down to a minimum and lessens the chance of crashes and errors etc..

**Reply by jadasse** [reply]
I agree

**Comment by MrPepka** [comment]
It would be nice to see the emulation of Aureal Vortex sound cards, but the documentation for those cards is probably lost media :(
I am also curious if the 86box will (in the distant future) emulate Pentium III, IV and newer CPUs together with motherboards, will it emulate AC'97 (and maybe even HD Audio) sound cards?

**Reply by Cacodemon345** [reply]
It will remain hardcapped to Pentium II/Celeron Mendocino machines.

**Reply by ArnoldUK** [reply]
I dont see the point wanting to emulate a Pentium IV or later CPU and motherboards ? It's called 86Box for a reason.

**Reply by jadasse** [reply]
> I dont see the point wanting to emulate a Pentium IV or later CPU and motherboards ? It's called 86Box for a reason.

And they have already established that emulating Pentium III at a perfect 100% 24/7 would not be feasible on today's hardware.

**Comment by Exemptus** [comment]
My hope is for even more accuracy in hardware emulation and quirks; for instance, 8087 and other chips. I'm less interested in exotic machines, and I hope one day the community will produce a single-clock accurate emulator of common PC hardware elements, with configurable levels of accuracy for less capable hosts.

**Reply by ArnoldUK** [reply]
I'd like to see an optimized win32 only version without all the QT baggage and a Windows API interface. Hardware DirectX and SDL (no OpenGL) rendering. More accurate mouse control with configurable hotkey for mouse capture/release.

**Reply by jriwanek** [reply]
We already have that, It's not currently available from the jenkins, but you can grab non-QT win32 builds from the github actions build.

**Comment by NanaMizukiAnother7** [comment]
For me I'd like to see the emulation of Promise Ultra33, Ultra66, EIDE 4030+, and DC4030VL-2.

But unfortunately, there are no datasheets of these controllers yet.

Related: #2479

**Comment by headsetguy37** [comment]
With ICH2 getting its own feature branch, I'm hoping for the original ICH. (Seems a bit backwards to me to do ICH2 before ICH, but whatever...)

I'd also love to see the Packard Bell PB600 "Agoura" motherboard, but it uses a non-standard chipset for which there is no datasheet (VLSI VL82C590 "Supercore"), so it's undoable at this point. I also don't know of any other boards that used that chipset.

**Comment by guest123guest** [comment]
Are there any plans for a version dedicated towards embedded installations without a GUI?

**Comment by MyBlueHorizon** [comment]
I hope 86box will have optional emulate accuracy so that user can get balance speed and software compatibility which decide on their need.

**Reply by TC1995** [reply]
The purpose of this emulator is to have accuracy at the maximum level.

**Comment by dhrdlicka** [comment]
Please raise further feature requests as separate issues or discussions.
